.github/actions/setup_rye/action.yml

```diff
@@ -14,10 +14,6 @@ runs:
       run: |
         echo "MATURIN_PEP517_ARGS=--zig" >> $GITHUB_ENV
       shell: bash
-    - name: Set UV_CACHE_DIR for MacOS
-      if: runner.os == 'macOS'
-      run: echo "UV_CACHE_DIR=/Users/gh-runner/Library/Caches/uv" >> $GITHUB_ENV
-      shell: bash
     - name: Set UV_CACHE_DIR for Windows
```

Comment:
The other cache settings are also not required for the temporary public runners. The containers are destroyed after the run anyway.